nautobot/ui/package.json

```diff
@@ -62,13 +106,19 @@
     },
     "devDependencies": {
         "@chakra-ui/cli": "^2.3.0",
-        "@craco/craco": "^7.1.0",
+        "babel-jest": "^29.6.2",
+        "babel-plugin-module-resolver": "^5.0.0",
         "eslint": "^8.44.0",
         "eslint-config-react-app": "^7.0.1",
         "markdownlint-cli": "^0.35.0",
         "prettier": "^2.8.8",
         "react-app-alias": "^2.2.2",
         "react-app-alias-ex": "^2.1.0",
```

Comment:
It seems like you're no longer using these libraries (it appears you have your own alias tool) so these could probably be deleted.